lib/insert_opts.rb

```diff
@@ -72,12 +72,6 @@ def initialize(
   # given job kind, a single instance is allowed for each combination of args
   # and queues. If either args or queue is changed on a new job, it's allowed to
   # be inserted as a new job.
-  #
-  # Uniquenes is checked at insert time by taking a Postgres advisory lock,
-  # doing a look up for an equivalent row, and inserting only if none was found.
-  # There's no database-level mechanism that guarantees jobs stay unique, so if
-  # an equivalent row is inserted out of band (or batch inserted, where a unique
-  # check doesn't occur), it's conceivable that duplicates could coexist.
```

Comment:
I could've changed this to talk about `unique_key`, but the paragraph was essentially a block warning not to insert unique jobs out of band, so felt like it wasn't really needed anymore.